### ISSUE TITLE
Reversal API

### ISSUE THREAD

**Issue by mohamedsaeedstc** [camaraproject/CarrierBillingCheckOut#115]
Hello,
In addition to the refund API, it's recommended to have reversal API, which can be used as a correction action from "merchant" side in case of any issue is happened at their side.
The main difference between reverse and refund:
1- Reversal will return the money "pre-paid" or cancel bill record without appearing in customer movements.
2- Reversal will return Fees, Commission and everything without charging merchant or customer anything
3- Reversal will be allowed for a certain duration using original transaction reference "Payment Id"

Refund can be used to return money for customer, even after days of the original transaction and no need to have the original transaction reference "Payment Id"

Thanks
Mohamed

**Comment by PedroDiez** [comment]
Agreed to have aligment and Business guidance from GSMA OGW Product first

**Comment by PedroDiez** [comment]
As commented in 20/DEC meeting:
- Every Operator to check internally priority of this feature
- Also welcome Use Cases to understand requirements

**Comment by PedroDiez** [comment]
WG Output: Out of scope for MetaRelease Spring 25
